.travis.yml

```diff
@@ -16,6 +16,7 @@ python:
   - 3.4
   - 3.5
   - 3.6
+  - 3.7
   - pypy
```

Comment:
You may find that switching to xenial globally will mean pypy doesn't show up. I had to [do this](https://github.com/cdent/pastescript/blob/2942a43f1a89d89ffd6d4fbb926f47d812cecd6d/.travis.yml#L22) somewhere else.

Otherwise, lgtm, assuming other tests pass.

Reply:
yeah, looks like you need the change I indicated.

Reply:
@cdent Thank you for your hint, now all tests are green. 😃